sno/structure.py

```diff
@@ -696,6 +696,24 @@ def get_feature_path(self, pk):
         ).hexdigest()  # hash to randomly spread filenames
         return "/".join([".sno-table", pk_hash[:2], pk_hash[2:4], pk_enc])
 
+    _FEATURE_PATH_PATTERN = re.compile(r"^.sno-table/../../(?P<encoded>.+)$")
+
+    _FEATURE_PATH_PATTERN = re.compile(r"^.sno-table/meta/../(?P<encoded>.+)$")
```

Comment:
are either of these used? (can't see any usages) they're both called `_FEATURE_PATH_PATTERN`, though one's actually about meta paths

Reply:
oops. Removed